pages/summer-of-food-and-fun.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react'
+import { NextPage, GetStaticProps } from 'next'
+
+import { Service, getServices } from '../cms/services'
+import { CardList } from '../src/Components/CardList'
+import { Layout } from '../src/Components/Layout'
+
+interface ListPageProps {
+  services: Array<ServicePreview>
+}
+
+export type ServicePreview = Pick<
+  Service,
+  | 'id'
+  | 'organisation'
+  | 'title'
+  | 'shortDescription'
+  | 'image'
+  | 'costValue'
+  | 'costQualifier'
+  | 'age'
+  | 'categories'
+  | 'interests'
+  | 'feelings'
+  | 'gender'
+  | 'format'
+  | 'score'
+  | 'promoted'
+  | 'area'
+>
+
+export const ListPage: NextPage<ListPageProps> = ({ services }) => {
+  // Default to showing the welcome screen
+  const [showWelcome, setShowWelcome] = useState<boolean | undefined>()
+
+  useEffect(() => {
+    // Attempt to retrieve the showWelcome value from local storage
+    const stored = window.localStorage.getItem('showWelcome')
+
+    // If it's been correctly set to false, update state
+    stored === 'false' ? setShowWelcome(false) : setShowWelcome(true)
+  }, [showWelcome])
+
+  if (showWelcome === undefined) {
+    return <></>
+  }
+
+  return (
+    <>
+      <Layout>
+        <CardList services={services} listType="filtered" />
+      </Layout>
+    </>
+  )
+}
+
+export default ListPage
+
+export const getStaticProps: GetStaticProps = async () => {
+  const allServices = getServices()
+
+  const allServicePreviews: Array<ServicePreview> = allServices
+    .filter((service) => service.provider === 'Summer of Food and Fun 2022')
```

Comment:
I think we want to make this filter if there is any provider string here that isn't empty. There will be the two other programmes that won't match 'Summer of Food and Fun 2022'